.gitignore

```diff
@@ -6,6 +6,7 @@
 .pnp.js
 .yarn/install-state.gz
 .idea
+.github
```

Comment:
If you added .github to the gitignore file you should remove this folder in the github.

Reply:
done